src/commands/dev/lib/dev-builder.ts

```diff
@@ -116,15 +116,15 @@ export async function executeBuild(
   let result: BuildResult;
 
   let { buildProcess } = match;
-  if (!buildProcess) {
+  if (!runInProcess && !buildProcess) {
     devServer.output.debug(`Creating build process for ${entrypoint}`);
     buildProcess = await createBuildProcess(
       match,
       devServer.buildEnv,
       devServer.output
     );
 
-    if (process.stdout.isTTY) {
+    if (!devServer.debug) {
```

Comment:
We don't need to check isTTY anymore?

Reply:
It makes more sense to me to show the logs when `--debug` is used, instead of basing it on whether or not stdout is a TTY (When would it not be when using `now dev`? You should be piping it to anything).